contrib/dockerd-rootless.sh

```diff
@@ -54,6 +54,27 @@ if ! [ -d "$HOME" ]; then
 	exit 1
 fi
 
+mount_directory() {
+	if [ -z "$_DOCKERD_ROOTLESS_CHILD" ]; then
+		echo "mount_directory should be called from the child context. Otherwise data loss is at risk"
+		exit 1
+	fi
+
+	local directory="$1"
+	local mount_options="${2:---bind}"
```

Comment:
Just noticed the use of `local` here, which I think is not part of POSIX, and maybe incorrect for the shebang used (`#!/bin/sh`)

Reply:
Oh, great catch. I was unaware of that. Will update the PR.

Reply:
Thanks! It will _probably_ work in many cases, but the intent of some of these scripts was to keep them as portable as possible (and don't depend on bashisms); some distros may be stricter than others for `/bin/sh`, so it could fail. And _probably_ for this one, it's not critical to have.

Reply:
Updated! Thanks for the catch! I have tested that it works as expected.